android/app/build.gradle

```diff
@@ -176,7 +176,7 @@ android {
         applicationId "me.rainbow"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 12
+        versionCode 14
```

Comment:
Why did u skip 13?

Reply:
I had uploaded 13, found the hot-fix regression and it wouldnt let me reupload it even though I had removed